app/assets/stylesheets/_home.scss

```diff
@@ -0,0 +1,7 @@
+#welcome {
+	margin-top: 1em;
+}
+
+body {
```

Comment:
This is going to apply to _all_ views - we should be more specific with CSS selectors.